.github/workflows/ci-pulsar-website-next-build.yaml

```diff
@@ -33,9 +33,9 @@ jobs:
     timeout-minutes: 600
     steps:
       - name: checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
-          fetch-depth: 2
+          fetch-depth: 1
```

Comment:
here, we need depth 2, we need to compare the changes of the last 2 commits because we judge which versions of the documents have changed and only build the changed versions

Reply:
Thanks! Reverted.